templates/manifest.json

```diff
@@ -0,0 +1,61 @@
+{% load webpack_static from webpack_loader %}
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "{{ name }}",
+  "description": "{{ description }}",
+  "short_name": "{{ short_name }}",
+  "icons": [
+    {
+      "src": "{% webpack_static "favicons/android-chrome-36x36.png" %}",
+      "sizes": "36x36",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-48x48.png" %}",
+      "sizes": "48x48",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-72x72.png" %}",
+      "sizes": "72x72",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-96x96.png" %}",
+      "sizes": "96x96",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-144x144.png" %}",
+      "sizes": "144x144",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-192x192.png" %}",
+      "sizes": "192x192",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-256x256.png" %}",
+      "sizes": "256x256",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-384x384.png" %}",
+      "sizes": "384x384",
+      "type": "image/png"
+    },
+    {
+      "src": "{% webpack_static "favicons/android-chrome-512x512.png" %}",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "scope": "/",
+  "start_url": "{% url 'home' %}",
+  "display": "fullscreen",
```

Comment:
Is this a valid value? Shouldn't it be `standalone`?

Reply:
> fullscreen: All of the available display area is used and no user agent chrome is shown. 

It also fallbacks to `standalone` if `fullscreen` cannot be used.

Source: https://developer.mozilla.org/en-US/docs/Web/Manifest